src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/Providers/AbstractSemanticQuickInfoProvider.cs

```diff
@@ -104,10 +104,10 @@ internal abstract partial class AbstractSemanticQuickInfoProvider : AbstractQuic
 
             // We calculate the set of supported projects
             candidateResults.Remove(bestBinding);
-
             foreach (var candidate in candidateResults)
             {
-                if (!candidate.Item3.SequenceEqual(bestBinding.Item3, LinkedFilesSymbolEquivalenceComparer.Instance))
+                // Does the candidate have anything remotely equivalent?
```

Comment:
Nit: On first reading of this comment I thought it meant that you actually loosened the equivalence check on individual items ("anything *remotely* equivalent"). But I think you meant for the emphasis to be on the *anything* instead, right? If so, consider getting rid of the word "remotely" there.